languages/haskell/exercises/shared/.docs/cli.md

```diff
@@ -0,0 +1,8 @@
+# CLI
+
+You can run the tests by opening a command prompt in the exercise's directory, and then running the [`stack test` command][docs-stack-test]. If your code compiles this command will show you which tests passed and which failed - when a test fails the rest might not be executed so just focus on making tests pass one by one.
```

Comment:
`when a test fails the rest might not be executed so just focus on making tests pass one by one.`
I feel like this wording is a bit confusing.

Reply:
I agree. 
I was trying to convey that I haven't seen disabled tests in Haskell. Rather  a `configFailFast` flag that if I understand right stops executing tests as soon as one of them fails? But it can look like there is just one test and that might be confusing. Can you check out the change I pushed?

Reply:
Looks good!